docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc

```diff
@@ -0,0 +1,28 @@
+[[dcb-requisition]]
+= Create Requisition for Device Configuration Backup
+
+After you have xref:operation:device-config-backup/ssh.adoc#backup-script[created a backup script for each device type] and xref:operation:device-config-backup/configuration.adoc#poller-config[configured the TFTP port], you can create a requisition for each device whose configuration you want to back up.
+
+To activate the device configuration backup, you must add metadata that defines the device configuration backup script you created for the node's device type, and the device's username and password.
+
+. Log in to OpenNMS and click the gears icon.
+. Choose *Provisioning>Manage Provisioning Requisitions*.
+. Click *Add Requisition* and enter a name for the requisition.
```

Comment:
Something to this effect to let users know they don't have to start with fresh nodes.
```suggestion
. Click *Add Requisition* and enter a name for the requisition.
If you want to enable device config backup on existing nodes in a requisition, select the desired requisition and edit the node definition.
```